notebooks/tensorflow/getting-started-keras.ipynb

```diff
@@ -1375,32 +1399,11 @@
         }
       ]
     },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "J2WGqwAzc3xM",
-        "colab_type": "text"
-      },
-      "source": [
-        "### Hyperparameter tuning\n",
-        "\n",
-        "You can optionally perform hyperparameter tuning by using the included\n",
-        "`hptuning_config.yaml` configuration file. This file tells AI Platform to tune the batch size and learning rate for training over multiple trials to maximize accuracy.\n",
-        "\n",
-        "In this example, the training code uses a [TensorBoard\n",
-        "callback](https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/TensorBoard),\n",
-        "which [creates TensorFlow `Summary`\n",
-        "`Event`s](https://www.tensorflow.org/api_docs/python/tf/summary/FileWriter#add_summary)\n",
-        "during training. AI Platform uses these events to track the metric you want to\n",
-        "optimize. Learn more about [hyperparameter tuning in\n",
-        "AI Platform Training](https://cloud.google.com/ml-engine/docs/tensorflow/hyperparameter-tuning-overview)."
-      ]
-    },
     {
```

Comment:
I think this cell is partially duplicated? See https://colab.research.google.com/github/GoogleCloudPlatform/cloudml-samples/blob/e88c0b6a7e6ed7bf4168a9de62cf9c42c4256c72/notebooks/tensorflow/getting-started-keras.ipynb#scrollTo=J2WGqwAzc3xM

I would delete lines 1399 to 1419.

Reply:
![duplicate-cell](https://user-images.githubusercontent.com/1772874/74174871-99eb0180-4be9-11ea-8ab9-15c71d450213.png)

Reply:
Did modification, removed cell. Thanks @alecglassford